### ISSUE TITLE
Conditional ternary expressions not fully evaluating nested method calls

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#64]
The following code will not compiler

```
player_potion1 := @player->GetPotion1() = Nil ? @player->GetPotion1()->ToString() : "None";
player_potion2 := @player->GetPotion2() = Nil ? "None" : @player->GetPotion2()->ToString();```

**Comment by objeck** [comment]
Refer to programs/test/regress/regress1.obs

**Comment by objeck** [comment]
Fixed and released in 87a1efba70caa1eb90e3308c1c12027958aaafc9
